emails/components/NamespaceDetails.tsx

```diff
@@ -0,0 +1,28 @@
+import Image from 'next/image';
+import Logo from '../assets/logo.png';
+import { Button, Heading, Img, Link, Text } from '@react-email/components';
+import { PrivateCloudCreateRequestBody } from '@/schema';
+
+export default function NamespaceDetails({ cluster, licencePlate }: { cluster: any; licencePlate?: any }) {
```

Comment:
Let's try to avoid using `any` type wherever possible. I believe both arguments are of type `string`, correct?